tests/automation_framework/globals.py

```diff
@@ -13,3 +59,14 @@
 event_provider = "http://10.66.241.55:22011"
 
 blockchain_type = ''
+
+# Direct test mode = listener or client_sdk
+direct_test_mode = "sdk"
```

Comment:
I think we should rename this to just test_mode? Because the SDK mode also applies to proxy_model, we can decide to take this up in the next commit.